src/TokenFuzzTests.sol

```diff
@@ -0,0 +1,165 @@
+// SPDX-FileCopyrightText: © 2023 Dai Foundation <www.daifoundation.org>
+// SPDX-License-Identifier: AGPL-3.0-or-later
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+pragma solidity >=0.8.0;
+
+import "./TokenTests.sol";
+import "./TokenFuzzChecks.sol";
+
+abstract contract TokenFuzzTests is TokenTests, TokenFuzzChecks {
+
+
+    // ************************************************************************************************************
+    // Mint/Burn
+    // ************************************************************************************************************
+
+    function test_fuzzMint(
+        address who,
+        uint256 mintAmount
+    ) public {
+        assertVarsSet();
+        fuzzCheckMint(_token_, _contractName_, who, mintAmount);
+    }
+    function test_fuzzBurn(
+        address who,
+        uint256 mintAmount,
+        uint256 burnAmount
+    ) public {
+        assertVarsSet();
+        fuzzCheckBurn(_token_, who, mintAmount, burnAmount);
+    }
+    function test_fuzzCheckBurnInsufficientBalance(
+        address to,
+        uint256 mintAmount,
+        uint256 burnAmount
+    ) public {
+        assertVarsSet();
+        fuzzCheckBurnInsufficientBalance(_token_, _contractName_, to, mintAmount, burnAmount);
+    }
+    function test_fuzzCheckTokenModifiers(
+        address sender
+    ) public {
+        assertVarsSet();
+        fuzzCheckTokenModifiers(_token_, _contractName_, sender);
+    }
+
+    // ************************************************************************************************************
+    // ERC20
+    // ************************************************************************************************************
+    
+    function test_fuzzMetadata(
+    ) public {
+        assertVarsSet();
+        fuzzCheckMetadata(_token_, _tokenName_, _symbol_, _version_, _decimals_);
+    }
```

Comment:
Does it make sense? I don't see anything being fuzzed here.

Reply:
Good catch, removed.